Makefile

```diff
@@ -176,7 +176,7 @@ install: default
 
 ### test:             Run the test case
 test:
-	prove -I../test-nginx/lib -I./ -r -s t/
+	TEST_NGINX_BINARY=`which openresty` prove -I../test-nginx/lib -I../test-nginx/inc -I./ -r -s t/
```

Comment:
why change this line? which is non-related with heartbeat plugin.

Reply:
Reverted in new commit.